wikimedia/wikipedia-ios
# T376071 intro screen

**Phabricator:** 
https://phabricator.wikimedia.org/T376071

### Notes
* 

### Test Steps
Turn YIR back on by uncommenting out the code in WMFProfileViewModel, lines 163-174
Navigate to YIR by clicking profile, then year in review
Confirm the first slide is visible

### Screenshots/Videos
![Simulator Screenshot - iPhone 15 Pro - 2024-10-09 at 12 39 09](https://github.com/user-attachments/assets/04cb9ee6-eaf9-4488-b90a-96e6329e3a2d)
![Simulator Screenshot - iPhone 15 Pro - 2024-10-09 at 12 39 27](https://github.com/user-attachments/assets/05b7ad67-dc83-4fea-bdb9-f587cd3874bc)
![Simulator Screenshot - iPhone 15 Pro - 2024-10-09 at 12 40 00](https://github.com/user-attachments/assets/7c95913f-9eab-43d6-8266-c822b498f548)
![Simulator Screenshot - iPhone 15 Pro - 2024-10-09 at 12 40 15](https://github.com/user-attachments/assets/8cc33674-cf1b-4bc8-b7e8-a856f8cc0454)
